src/plugins/intel_npu/tools/single-image-test/main.cpp

```diff
@@ -127,6 +127,7 @@ DEFINE_string(
     ref_dir,
     "",
     "A directory with reference blobs to compare with in run_test mode. Leave it empty to use the current folder.");
+DEFINE_string(ref_results, "", "Reference files used during 'run_test'");
```

Comment:
Please, specify input format in case of several inputs.
Is it `--ref_results "file1;file2"`?

Are relative paths supported? e.g. `--ref_dir /part/of/path/ --ref_results rest/of/path/file.blob`? 

Are absolute paths supported? e.g. `--ref_results /abs/path/file.blob`